build_deps/requirements.txt

```diff
@@ -1,2 +1,3 @@
 # TensorFlow greater than this date is manylinux2010 compliant
 tf-nightly-2.0-preview>=2.0.0.dev20190802
+sklearn
```

Comment:
At first this was concerning, but the requirements files are only used in our builds/tests and are not used as requirements in `setup.py` so I'm fine with this.